### DISCUSSION TITLE
Cannot edit comments on issues

### DISCUSSION THREAD

**Discussion by kordwarshuis** [community/community#131859] [Projects and Issues] [answered]
### Select Topic Area

Question

### Body

I cannot edit comments on issues that I made.

Tested in different browsers.
Tested with two accounts.

I created a repo with a test account, created an issue with this account, created a comment with this account.

I used to be able to click on the three dots at the right top of an issue and edit a comment. Now nothing happens.

Anyone else having this problem?

What am I doing wrong?

~Kor

**Comment by ebndev** [comment]
Hi @kordwarshuis, thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our Projects and Issues category, as this topic might be more relevant there! If you could provide a screenshot, that might help someone give you a nudge in the right direction.

Best of luck!

**Comment by kordwarshuis** [comment]
![_My Video](https://github.com/community/community/assets/2950583/6612496d-5434-41f4-bf47-86ea3e9caea2)

Here you see nothing happens. The same for the comment one block lower.

**Comment by kordwarshuis** [comment]
New insight: The problem is in the web page / web app; on iOS app editing works as normal.

**Comment by kordwarshuis** [comment, marked as answer]
This was a problem in several browsers. But now after repeated testing, the issue seems to be a Brave Browser issue (Version 1.67.123 Chromium: 126.0.6478.126 (Official Build) (arm64)).

**Reply by samus-aran** [reply]
Thanks @kordwarshuis for coming back and adding the additional information. I'll mark is as the answer to help other folks in the same situation.

**Comment by kordwarshuis** [comment]
It seemed to be a Brave browser issue. Disabling the extension solved the problem.

In the end it was a problem in a browser extension written by myself, injecting Bootstrap in every site, without user interaction… Disabling the extension and then fixing and updating the extension fixed it…
